src/amo/components/GuidesAddonCard/styles.scss

```diff
@@ -0,0 +1,99 @@
+@import '~amo/css/styles';
+
+.GuidesAddonCard {
+  display: flex;
+  flex-direction: column;
+
+  // stylelint-disable max-line-length
+  // TODO: check on get-firefox button style.
+  // See https://github.com/mozilla/addons-frontend/issues/6432#issuecomment-433083079.
```

Comment:
It would be more helpful if this linked to https://github.com/mozilla/addons-frontend/issues/6916 but, really, I think you can remove these comments. You already have one todo comment about it.